Makefile

```diff
@@ -13,4 +13,4 @@ test:
 
 lint:
 	@echo "$(OK_COLOR)==> Linting with golangci-lint$(NO_COLOR)"
-	@docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.35.2 golangci-lint run -v
+	@docker run --rm -v $(PWD):/app -w /app golangci/golangci-lint golangci-lint run -v
```

Comment:
better pin it to some fixed version, because otherwise it will be using `latest` that is stored locally, so it may be very very outdated - if I pulled `v1.20` when it was latest then I will always use it until I'll pull other latest explicitly

Reply:
I'll lock the version on the action as well then